jobs/promptLicenceCreation.ts

```diff
@@ -10,38 +10,20 @@ import { ManagedCase } from '../server/@types/managedCase'
 import LicenceStatus from '../server/enumeration/licenceStatus'
 import { EmailContact } from '../server/@types/licenceApiClientTypes'
 import { convertToTitleCase } from '../server/utils/utils'
-import Container from '../server/services/container'
+import PromptLicenceCreationService from './promptLicenceCreationService'
 
 initialiseAppInsights()
 buildAppInsightsClient('create-and-vary-a-licence-prompt-licence-create-job')
 
 const { caseloadService, prisonerService, communityService, licenceService } = services
 
-const pollPrisonersDueForLicence = async (
-  earliestReleaseDate: Date,
-  latestReleaseDate: Date,
-  licenceStatus: LicenceStatus[]
-): Promise<ManagedCase[]> => {
-  const prisonCodes = config.rollout.prisons
-
-  return prisonerService
-    .searchPrisonersByReleaseDate(earliestReleaseDate, latestReleaseDate, prisonCodes)
-    .then(prisoners => prisoners.filter(offender => offender.status && offender.status.startsWith('ACTIVE')))
-    .then(caseload => new Container(caseload))
-    .then(caseload => caseloadService.pairNomisRecordsWithDelius(caseload))
-    .then(caseload => caseloadService.filterOffendersEligibleForLicence(caseload))
-    .then(prisoners => caseloadService.mapOffendersToLicences(prisoners))
-    .then(caseload => caseload.unwrap())
-    .then(prisoners =>
-      prisoners.filter(offender => licenceStatus.some(status => offender.licences.find(l => l.status === status)))
-    )
-}
+const promptLicenceCreationService = new PromptLicenceCreationService(prisonerService, caseloadService)
 
 const buildEmailGroups = async (
-  initialPromptCases: ManagedCase[],
-  urgentPromptCases: ManagedCase[]
+  urgentPromptCases: ManagedCase[],
+  initialPromptCases: ManagedCase[]
 ): Promise<EmailContact[]> => {
-  const managedCases = [...initialPromptCases, ...urgentPromptCases]
+  const managedCases = [...urgentPromptCases, ...initialPromptCases]
```

Comment:
probably doesn't matter but don't think this needed to change

Reply:
I just put them in the same order just so the code flows nicely. I know it doesn't make any actual diff as the all the values just get added to the array .

Reply:
yes, that makes sense